File: MiniDOMOffice/CRM/Interazioni/Telegrammi/CTelegrammiCursor.cs
using System;
using System.Collections;
using System.Collections.Generic;
using DMD;
using DMD.XML;
using DMD.Databases;
using DMD.Databases.Collections;
using minidom;
using minidom.repositories;
using static minidom.Sistema;
using static minidom.Anagrafica;
using static minidom.CustomerCalls;

namespace minidom
{
    public partial class CustomerCalls
    {

        /// <summary>
        /// Cursore sulla tabella dei telegrammi
        /// </summary>
        /// <remarks></remarks>
        [Serializable]
        public class CTelegrammiCursor 
            : CContattoUtenteCursor 
        {

            /// <summary>
            /// Costruttore
            /// </summary>
            public CTelegrammiCursor()
            {
                base.ClassName.Value = "CTelegramma";
            }

            private new object ClassName
            {
                get
                {
                    return null;
                }
            }

            /// <summary>
            /// Istanzia un nuovo oggetto
            /// </summary>
            /// <param name="dbRis"></param>
            /// <returns></returns>
            public override CContattoUtente InstantiateNewT(DBReader dbRis)
            {
                return new CTelegramma();
            }

            /// <summary>
            /// Restituisce l'oggetto corrente
            /// </summary>
            public new CTelegramma Item
            {
                get
                {
                    return (CTelegramma)base.Item;
                }

                set
                {
                    base.Item = value;
                }
            }

            /// <summary>
            /// Aggiunge una nuova riga
            /// </summary>
            /// <returns></returns>
            public new CTelegramma Add()
            {
                return (CTelegramma)base.Add();
            }
        }
    }
}

File: MiniDOMOffice/CRM/Interazioni/Telegrammi/CTelegrammiCursor.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema

Imports minidom.Anagrafica
Imports minidom.CustomerCalls



Partial Public Class CustomerCalls

    ''' <summary>
    ''' Cursore sulla tabella dei telegrammi
    ''' </summary>
    ''' <remarks></remarks>
    Public Class CTelegrammiCursor
        Inherits CCustomerCallsCursor

    
        Public Sub New()
            MyBase.ClassName.Value = "CTelegramma"
        End Sub

        Public Overrides Function InstantiateNew(dbRis As DBReader) As Object
            Return New CTelegramma
        End Function


        Public Shadows Property Item As CTelegramma
            Get
                Return MyBase.Item
            End Get
            Set(value As CTelegramma)
                MyBase.Item = value
            End Set
        End Property

         

    End Class


End Class

